packages/react-charting/src/utilities/ChartHoverCard/ChartHoverCard.base.tsx

```diff
@@ -1,7 +1,23 @@
 import * as React from 'react';
 import { IChartHoverCardStyles, IChartHoverCardStyleProps, IChartHoverCardProps } from './ChartHoverCard.types';
 import { classNamesFunction, IProcessedStyleSet } from '@fluentui/react';
-import { convertToLocaleString } from '../index';
+
+type LocaleStringDataProps = number | string | Date | undefined;
+const convertToLocaleString = (data: LocaleStringDataProps, culture?: string): LocaleStringDataProps => {
```

Comment:
why duplicating code

Reply:
The difference between importing utilities and without utilities are as follows:
![image](https://github.com/microsoft/fluentui/assets/120183316/9d18311c-b93e-48ee-bd34-ce203dc9016d)

Reply:
There shouldn't be code duplication. Find another way to fix this bloat

Reply:
done